crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,25 @@
+//! Asset processing in Bevy is a framework for automatically transforming artist-authored assets into the format that best suits the needs of your particular game.
+//!
+//! You can think of the asset processing system as a "build system" for assets.
+//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified processing steps on the asset.
+//! This can include things like converting a `.psd` file to a `.png`, compressing a `.wav` file to an `.ogg`, or generating mipmaps for a texture.
```

Comment:
Can processing currently do this? When I looked into it getting a process to change the filename/extension required a lot of changes to asset processing. I'd love to be wrong about that, though.

Reply:
For the first example, I think its better to call out a more common gamedev asset pipeline, such as converting a `.png` into a compressed GPU-friendly texture format. In the context of asset processing, `png` isn't often going to be the target format.